server/controllers/list.js

```diff
@@ -1003,6 +1003,9 @@ const updateListItem = (req, resp) => {
           : ActivityType.ITEM_RESTORE;
       }
 
+      itemToUpdate.editedBy = userId;
+      editedItemActivity = ActivityType.ITEM_EDIT;
```

Comment:
`editedItemActivity` will always be `ActivityType.ITEM_EDIT` - is this expected?

Reply:
This activityType should never be registered as EDITING means edition of name or description. And we're already saving those activities, so this one doesn't make any sense.

I removed this one